requirements.txt

```diff
@@ -0,0 +1,2 @@
+nose==1.3.7
+requests==2.28.2
```

Comment:
I figure I should either remove the `nose` requirement and just mention to install it in the README if you want to run tests, or split out requirements, but it seems premature to split out requirements.

Reply:
(this now applies to `freezegun` as well)